README.md

```diff
@@ -184,11 +184,23 @@ See [d3-ease](https://github.com/d3/d3-ease).
 
 <a name="label" href="#label">#</a> flamegraph.<b>label</b>(<i>[function]</i>)
 
-Adds a function that returns a formatted label. Example:
+Adds a function that returns a formatted label.
+
+Example:
 
 ```js
 flamegraph.label(function(d) {
-    return "name: " + d.name + ", value: " + d.value;
+    return "name: " + d.data.account;
```

Comment:
Since `.account` or `.accountID` are not required in the data structure, it might be better to add a second example with a custom field, making it clear it is a custom data structure, with a JSON example, so users don't run into issues when copy/paste the example.